src/Assets/AssetContainer.php

```diff
@@ -74,7 +82,16 @@ public function save()
 
         $model->save();
 
-        $this->model($model->fresh());
+        $this->model($model->fresh())
```

Comment:
Feels like this should be calling fromModel to avoid repetition?

Reply:
I know, but `fromModel` creates a new instance so we can't use that.
I could add a `fillFromModel` that both method could use.